src/protocols/bdx/BdxTransferSession.h

```diff
@@ -97,6 +107,7 @@ class DLL_EXPORT TransferSession
             TransferAcceptData transferAcceptData;
             BlockData blockdata;
             StatusReportData statusData;
+            MessageTypeData msgTypeData;
```

Comment:
Is this used in practice anywhere outside the unit test?  I'm not obviously seeing it....

Reply:
It is - see `PrepareOutgoingMessageEvent()` in BdxTransferSession.cpp

Also this struct will be crucial to any application that uses `TransferSession` because it will need to pass the ProtocolId and MessageType to `ExchangeContext::SendMessage()`

Reply:
Ah, it's actually used as an arg to `AttachHeaderAndSend` looks like.  OK, that's the part I was missing.